linz2osm/data_dict/models.py

```diff
@@ -220,8 +228,12 @@ def deduce_geometry_type(self):
     def feature_limit(self):
```

Comment:
Can we comment on why these numbers are set as they are, for future understanding?